### PR TITLE
removed duplicate code for WSProbe issue #20122

### PR DESCRIPTION
**Pull request by poojadshende**

### CODE REVIEW

**Comment by johanandren** [comment]
Refs #20122

**Comment by johanandren** [comment]
Hi @poojadshende the thing to change was to fix the Scaladoc not to remove the method?

**Comment by poojadshende** [comment]
@johanandren sorry for wrong modification. I am not able to find Scaldoc to fix. Can you please, let me know the file name or location? So that I will work on it. Thanks!

**Comment by johanandren** [comment]
It is the Scaladoc of the method you removed `outProbe` - it should not be removed but corrected as it now incorrectly is a copy of the docs of `inProbe`.

**Comment by ktoso** [comment]
+1 to what @johanandren explained :)
We need that method :)

**Comment by poojadshende** [comment]
@johanandren @jrudolph What should be the description for outProbe? Thanks!

File: akka-http-testkit/src/main/scala/akka/http/scaladsl/testkit/WSProbe.scala [hunk 1]
```diff
@@ -88,11 +88,6 @@ trait WSProbe {
    */
   def inProbe: TestSubscriber.Probe[Message]
 
-  /**
-   * The underlying probe for the ingoing side of this probe. Can be used if the methods
-   * on WSProbe don't allow fine enough control over the message flow.
-   */
-  def outProbe: TestPublisher.Probe[Message]
```
Comment (ktoso) [inline akka-http-testkit/src/main/scala/akka/http/scaladsl/testkit/WSProbe.scala]: Please bring back this method and make the comment say:

```
The underlying probe for the outgoing side of this probe. Can be used if the methods
on WSProbe don't allow fine enough control over the message flow.
```

**Comment by ktoso** [comment]
I commented on the line what the comment should be, please amend the commit @poojadshende :)

**Comment by johanandren** [comment]
(Closed as the fix was done over in #20471) Please keep the work in one PR rather than opening a new one in the future @poojadshende, thanks!

**Pull request closed without merging**